app/views/accounts/_emails.html.erb

```diff
@@ -1,17 +1,8 @@
 <div class="clearfix border-bottom border-primary-light">
   <div class="col col-12 margin-bottom-1 margin-top-0">
-    <h3 class="h2 col col-6 margin-0">
+    <h3 class="h2 col col-12 margin-0">
```

Comment:
Since we're not arranging the heading and button anymore, we may not need any of the column classes anymore?

```suggestion
    <h3 class="h2 margin-0">
```